src/app/feedback/feedback-view.component.ts

```diff
@@ -110,16 +110,19 @@ export class FeedbackViewComponent implements OnInit, OnDestroy {
     );
   }
 
-  editTitle(mode) {
-    this.editTitleMode = mode;
+  changeEditTitle(mode) {
+    this.changeEditTitleMode = mode;
   }
 
   setTitle() {
     this.couchService.put(this.dbName + '/' + this.feedback._id, this.feedback).subscribe(
-      () => {
-        this.editTitleMode = false;
+      (res) => {
+        this.changeEditTitleMode = false;
+        this.couchService.put(this.dbName + '/' + res.rev, this.feedback).subscribe(
```

Comment:
There shouldn't be another request to CouchDB here.  You need to update `this.feedback._rev` to match `res.rev`

Reply:
Okay. I will fix this too. Thank you for the feedback here as well.

Reply:
@paulbert - I made the latest changes you requested.